content/docs/getting-started/updating-the-ui/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Updating the Backstage UI
+lastUpdated: '2021-08-06T12:00:00.0Z'
+description: How to configure the Backstage UI Layout
+---
+
+## Introduction
+
+Roadie allows you to configure which plugins you want to use via our UI. This guide explains the different types of
+plugin components and the different places they can be integrated into the app.
+
+## Prerequisites
+
+* You must be an admin to edit the UI layout. Please complete [this guide](/docs/getting-started/getting-started-for-admins/) before moving on to this one.
```

Comment:
This doesn't need to be a bullet, since there is just one bullet point.

```suggestion
You must be an admin to edit the UI layout. Please complete [this guide](/docs/getting-started/getting-started-for-admins/) before moving on to this one.
```